plotter/labels_test.go

```diff
@@ -5,12 +5,114 @@
 package plotter_test
 
 import (
+	"image/color"
 	"testing"
 
+	"gonum.org/v1/plot"
 	"gonum.org/v1/plot/cmpimg"
+	"gonum.org/v1/plot/plotter"
+	"gonum.org/v1/plot/vg"
+	"gonum.org/v1/plot/vg/draw"
 )
 
 func TestLabels(t *testing.T) {
 	cmpimg.CheckPlot(ExampleLabels, t, "labels.png")
 	cmpimg.CheckPlot(ExampleLabels_inCanvasCoordinates, t, "labels_cnv_coords.png")
 }
+
+func TestLabelsWithGlyphBoxes(t *testing.T) {
```

Comment:
It would be good to have some description of the things that this is exercising so that the expectations can be confirmed.